Code/Framework/AzManipulatorTestFramework/Include/AzManipulatorTestFramework/ImmediateModeActionDispatcher.h

```diff
@@ -47,6 +50,9 @@ namespace AzManipulatorTestFramework
         //! Get the current state of the keyboard modifiers.
         KeyboardModifiers GetKeyboardModifiers() const;
 
+        // EditorModifierKeyRequestBus overrides ...
+        KeyboardModifiers QueryKeyboardModifiers() override;
+
```

Comment:
It's redundant to have both `GetKeyboardModifiers` and `QueryKeyboardModifiers`, both public, both doing the same thing. To simplify the interface, I'd suggest to get rid of one, probably `GetKeyboardModifiers` since it's not inherited.

Reply:
That's a fair point, I\ll just delete `GetKeyboardModifiers` and use `QueryKeyboardModifiers` directly, will update shortly :+1: